rllib/models/torch/fcnet.py

```diff
@@ -13,36 +13,32 @@
 logger = logging.getLogger(__name__)
 
 
-class FullyConnectedNetwork(TorchModelV2, nn.Module):
+class FullyConnectedNetwork(TorchModelV2):
```

Comment:
Can you make sure to update the documentation to not use nn.Module either?

Reply:
sure ... +1

Reply:
done